ENTESB-11369: Errors in install scripts for fuse-online

phantomjinx opened the pull request:
* install_ocp.sh
 * Tests username/password credentials for redhat registry
 * Fixes secret not being created due to lack of email address - mandatory
   for pull-secret
 * Create secret AFTER possible deletion of project. Otherwise, we just
   delete the secret!

* *.yml
 * Redhat registry is insecure so image streams must be allowed to fetch
   images as http. Otherwise, streams report error
   'http: server gave HTTP response to HTTPS client.'

lgarciaaco commented on install_ocp.sh:
```diff
@@ -858,7 +858,19 @@ create_secret_if_not_present() {
-    local result=$(oc create secret docker-registry syndesis-pull-secret --docker-server=registry.redhat.io --docker-username=$username --docker-password=$password)
+
+    # Testing access that credentials are correct
+    local reply=$(curl -IsL -u ${username}:${password} "https://sso.redhat.com/auth/realms/rhcc/protocol/redhat-docker-v2/auth?service=docker-registry&client_id=curl&scope=repository:rhel:pull")
```
What if we put this into a `while true` with a counter, and we give the user the opportunity to re introduce the credentials 3 times instead of exiting with the first attempt.

phantomjinx commented:
> Files in `resources` are generated with `release.sh` using `templates` as input. You rather modify the files in `templates` **BUT** .. it is a security measurement not to enable `insecure: true`. People from QE know that they have to manually change to `insecure: true` to test with brew.

OK. Will remove from PR.

phantomjinx commented:
Removed 'insecure' yml changes.
Added in 3-try attempts for credentials
Added in wait and check on operator imagestream to avoid the wait_for_deployments hang
